consensus/dummy/dynamic_fees_test.go

```diff
@@ -200,7 +200,7 @@ func testDynamicFeesStaysWithinRange(t *testing.T, test test) {
 	for index, block := range blocks[1:] {
 		tc := params.TestChainConfig
 		tc.FeeConfig.MinBaseFee = params.TestMinBaseFee
-		nextExtraData, nextBaseFee, err := CalcBaseFee(params.TestChainConfig, header, block.timestamp)
+		nextExtraData, nextBaseFee, err := CalcBaseFee(params.TestChainConfig, params.TestChainConfig.FeeConfig, header, block.timestamp)
```

Comment:
Can we add a test case where the base fee is both below and above the allowed range after the update?